A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1294,6 +1302,51 @@ class FMCMainDisplay extends BaseAirliners {
         return this._routeReservedPercent;
     }
 
+    /**
+     * Sets new Cruise FL if all conditions good
+     * @param input {float} Altitude or FL
+     * @returns {boolean} input passed checks
+     */
+    trySetCruiseFl(input) {
```

Comment:
Why not use this method in INIT A?

Reply:
Because in INIT A you can enter every FL regardless of your flight state.
In flight you are limited by more factors.

Reply:
yes u are right, I will look at that

Reply:
done.